### ISSUE TITLE
Dynamic Pricing option Set Outside Valid Limits

### ISSUE THREAD

**Issue by ksmoore125** [laterpay/revenue-generator#96]
**TL;DR:** Update the Less than 10 articles, post length 0-250 characters, dynamic price to $1.99 Pay Now

------------------------------

**Steps to Reproduce**
- Set the merchant to publishing `Fewer than 10` posts per month
- Create a new paywall with the single article item set to dynamic pricing
- Publish

**Observed Result**
- The paywall is not visible on the site
- Upon further research, this is caused by the following error:
`{"jwt":{"purchase_options":[{"price":{"errors":[{"code":"api_error.pay_now_price_out_of_range","message":"Pay Now price must be between {min} and {max}.","params":{"min":"199","max":"100000"},"field":"price_amount"}]}},{}]}}`

**Fix**
Update the following dynamic price to 1.99 so that it can stay Pay Now:

Articles per Month | Article Word Count | Purchase Option | Price | Payment Model
-- | -- | -- | -- | --
Less than 10 | 0-250 | Single Article | **1.99** | Pay Now
